apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -117,22 +127,6 @@ render self@{ state, setState } logger searchView isPersonating =
           { creditCards: fromMaybe mempty $ state.activeUser <#> _.creditCards
           , logger: logger
           }
-   search =
-     Router.route
-       { exact: true
-       , path: Just "/sök"
-       , render:
-         \_ -> Helpers.classy DOM.div "mitt-konto--container clearfix"
-                 [ foldMap Elements.loadingIndicator state.loading
-                 , case state.activeUser /\ state.adminMode of
-                     Just user /\ true -> searchView
-                     _ -> Views.loginView self logger
-                 ]
-       }
    noMatchRoute =
      -- TODO: Use Redirect when supported!
-     Router.route
-       { exact: false
-       , path: Nothing
-       , render: const mittKontoView
-       }
\ No newline at end of file
+     routeElement false true Nothing $ Views.userView self logger
```

Comment:
Does this solve the problem that the @vapaj was talking about? I.e. unsopported routes don't get redirected to home.

Reply:
Not, it doesn't. I just wanted to refactor the routes list a bit as the where clauses had got pretty repetitive.